src/test-support/template.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
It seems to me that this is (or could be) _basically_ the same as the jscodeshift one but we run a different function for actually testing (jscodeshift runs `runInlineTest` and this one runs `runTemplateTest`).

I would prefer to refactor these together to avoid the duplication that exists right now, and to also ensure that both support the same feature set. At the moment, I _think_ this file is missing the ability to test passing different command line options (that the template transform can access with `getOptions`).

Reply:
Makes sense, done.